Library/Homebrew/cleanup.rb

```diff
@@ -36,10 +42,18 @@ def self.cleanup_logs
       end
     end
 
-    def self.cleanup_cellar
-      Formula.installed.each do |formula|
-        cleanup_formula formula
+    def self.cleanup_cellar(formulae = Formula.installed)
+      unremovable_kegs = []
+      formulae.each do |formula|
+        cleanup_formula formula, unremovable_kegs
```

Comment:
Feels weird to pass `unremovable_kegs` as an argument; what about an instance variable instead?